lib/okta/service.rb

```diff
@@ -65,14 +65,16 @@ def delete_grant(user_id, grant_id)
     def metadata(iss)
       proxied_iss = iss.gsub(Settings.oidc.issuer_prefix, Settings.oidc.base_api_url + 'oauth2')
       with_monitoring do
-        get_url_with_token(proxied_iss + '/.well-known/openid-configuration')
+        get(proxied_iss + '/.well-known/openid-configuration', {},
```

Comment:
Not a ruby expert.

It looks like the previous behavior returned a `Okta::Response` and this may not?

Reply:
Yeah, that response would not get wrapped in the `Okta::Response.new`.

It looks like the `Okta::Response` adds a `success?` helper method vs just the body and status code from a base `get`.  But for metadata and oidc_jwks_keys it isn't used.

We could consider updating `call_with_token` to instead include a boolean arg like `include_token`, and then add one more `define_method` at the bottom?

Reply:
I believe okta responses can wrap any response, so just adding a `Okta:Response:new` to the calls should suffice. I would not have an issue with creating a new method however.

Reply:
I think a new method will be preferable here.  Then we aren't subject to unexpected changes to the Common::Client::Base `get` method definition.  Plus we aren't building up the request headers in 2 different ways.